ui/src/app/applications/components/utils.tsx

```diff
@@ -987,23 +987,55 @@ export const OperationState = ({app, quiet}: {app: appModels.Application; quiet?
     );
 };
 
+function isPodInitializedConditionTrue(status: any): boolean {
+    for (const condition of status.conditions) {
```

Comment:
```suggestion
    for (const condition of status.conditions) {
        if (!status.conditions) {
            return false;
        }
```

I'm not sure how Babel will transpile this down, but in node I got a runtime error with a missing conditions field.

Reply:
I had overlooked the fact that a field might be missing in some cases. I've now moved the existence check logic before the loop.